### PR TITLE
Design Layout implemented

### PR DESCRIPTION
**Pull request by afvc**
- New design structure
- new sass files (more division)
- Far hide and show with focus
- Login and Sign Up merged on one page with modals instead of separated html
- New box page “deleted” and changed into a modal
- Settings extra text boxes with same logic as far
- 90% flex box -grid only flex box on box_list
- all website static in the moment
- responsiveness is resolved in principle
-  #mobile-first

### CODE REVIEW

File: boxes/templates/boxes/box_create.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h2 class="text-dark">Create New Box</h2>
                 </p>
                 <p>
                     {{ form.description.errors }}
-                    {{ form.description.label_tag }}
+                    {{ form.descyription.label_tag }}
```
Comment (dethos) [inline boxes/templates/boxes/box_create.html]: Typo here

Comment (afvc) [inline boxes/templates/boxes/box_create.html]: eu tenho quase a certeza que isso já foi resolvido, no meu html isso não aparece

Comment (afvc) [inline boxes/templates/boxes/box_create.html]: btw, o meu github for mac tem as cores trocadas do que aparentemente vocês costumam ter, não sei se afetará tb aqui no site.

Comment (dethos) [inline boxes/templates/boxes/box_create.html]: You might have not pushed the last changes to this github branch

Comment (afvc) [inline boxes/templates/boxes/box_create.html]: já vou ligar o servidor nesse branch e confirmar, mas tenho ideia que sim, tudo o resto que alterei estava lá já.

File: boxes/templates/boxes/box_list.html [hunk 2]
```diff
@@ -1,81 +1,189 @@
...
-    <script src="{% static 'javascripts/box_list.js' %}"></script>
+
+<script src="{% static 'javascripts/box_list.js' %}"></script>
+
+
+<script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
+<script src="{% static 'javascripts/box_create.js' %}"></script>
+
+
+</section>
```
Comment (dethos) [inline boxes/templates/boxes/box_list.html]: This section shouldn't be ended inside the `script`'s block. Probably you should end it after line 176, before ending the `content` block.

File: boxes/templates/boxes/box_create.html (hunk 1 above)
Comment (afvc) [inline boxes/templates/boxes/box_create.html]: sim, está tudo dentro dos conformes, tanto no site como no código. dunno
captura de ecra 2016-08-19 as 12 41 19

Comment (dethos) [inline boxes/templates/boxes/box_create.html]: You are looking at the wrong file. This one is `box_create.html`.

File: boxes/templates/boxes/box_list.html (hunk 2 above)
Comment (afvc) [inline boxes/templates/boxes/box_list.html]: ok, will do both changes and then do i do a new pull request or how do i proceed?

File: stylesheets/_modal-cópia.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
+     form {
```
Comment (dethos) [inline stylesheets/_modal-cópia.scss]: please keep the file names in English. based on the name this looks like a duplicated file.

Comment (afvc) [inline stylesheets/_modal-cópia.scss]: that file probably is an error. i do not have that in the style.scss being called. i do have _modal 
and _modalxs. in principle you can delete it

File: boxes/templates/boxes/box_list.html (hunk 2 above)
Comment (dethos) [inline boxes/templates/boxes/box_list.html]: Just push the changes to this branch and they will end up here.

File: stylesheets/_modal-cópia.scss (hunk 3 above)
Comment (dethos) [inline stylesheets/_modal-cópia.scss]: ok :+1: , please do it and push the changes.

**Pull request merged**